assets/img/passport_composition.svg

```diff
@@ -1 +1 @@
"/><use transform="matrix(0.0087890625,0,0,0.0087890625,21.603515625,0)" xlink:href="#X"/><use transform="matrix(0.0087890625,0,0,0.0087890625,32.4052734375,0)" xlink:href="#aQ"/><use transform="matrix(0.0087890625,0,0,0.0087890625,43.20703125,0)" xlink:href="#aG"/><use transform="matrix(0.0087890625,0,0,0.0087890625,54.0087890625,0)" xlink:href="#aO"/><use transform="matrix(0.0087890625,0,0,0.0087890625,64.810546875,0)" xlink:href="#V"/><use transform="matrix(0.0087890625,0,0,0.0087890625,75.6123046875,0)" xlink:href="#ai"/></g><path d="M631 20c-350 0-501-215-501-562 0-355 162-560 502-560 250 0 399 118 446 323l-192 14c-23-124-109-196-262-196-242 0-305 171-305 415 1 245 61 427 304 427 151 0 248-77 267-215l190 12C1039-107 883 20 631 20" id="aR"/><g id="z"><use transform="matrix(0.0087890625,0,0,0.0087890625,0,0)" xlink:href="#V"/><use transform="matrix(0.0087890625,0,0,0.0087890625,10.8017578125,0)" xlink:href="#ac"/><use transform="matrix(0.0087890625,0,0,0.0087890625,21.603515625,0)" xlink:href="#ad"/><use transform="matrix(0.0087890625,0,0,0.0087890625,32.4052734375,0)" xlink:href="#aG"/><use transform="matrix(0.0087890625,0,0,0.0087890625,43.20703125,0)" xlink:href="#ae"/><use transform="matrix(0.0087890625,0,0,0.0087890625,54.0087890625,0)" xlink:href="#aR"/><use transform="matrix(0.0087890625,0,0,0.0087890625,64.810546875,0)" xlink:href="#aO"/><use transform="matrix(0.0087890625,0,0,0.0087890625,75.6123046875,0)" xlink:href="#V"/><use transform="matrix(0.0087890625,0,0,0.0087890625,86.4140625,0)" xlink:href="#ai"/></g><g id="A"><use transform="matrix(0.0087890625,0,0,0.0087890625,0,0)" xlink:href="#aI"/><use transform="matrix(0.0087890625,0,0,0.0087890625,10.8017578125,0)" xlink:href="#aI"/><use transform="matrix(0.0087890625,0,0,0.0087890625,21.603515625,0)" xlink:href="#aI"/></g><path d="M24-248c120-7 223 5 221 122C244-46 201 0 124 0H24v-248zM76-40c74 7 117-18 117-86 0-67-45-88-117-82v168" id="aS"/><path d="M175 0L67-191c6 58 2 128 3 191H24v-248h59L193-55c-6-58-2-129-3-193h46V0h-61" id="aT"/><path d="M146-102V0H94v-102L6-248h54l60 105 60-105h54" id="aU"/><g id="B"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#T"/><use transform="matrix(0.05,0,0,0.05,5,0)" xlink:href="#aS"/><use transform="matrix(0.05,0,0,0.05,17.950000000000003,0)" xlink:href="#aM"/><use transform="matrix(0.05,0,0,0.05,29.950000000000003,0)" xlink:href="#aT"/><use transform="matrix(0.05,0,0,0.05,42.900000000000006,0)" xlink:href="#O"/><use transform="matrix(0.05,0,0,0.05,53.85,0)" xlink:href="#T"/><use transform="matrix(0.05,0,0,0.05,58.85,0)" xlink:href="#O"/><use transform="matrix(0.05,0,0,0.05,69.80000000000001,0)" xlink:href="#aU"/></g><g id="C"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#O"/><use transform="matrix(0.05,0,0,0.05,10.950000000000001,0)" xlink:href="#aU"/><use transform="matrix(0.05,0,0,0.05,22.950000000000003,0)" xlink:href="#J"/><use transform="matrix(0.05,0,0,0.05,34.95,0)" xlink:href="#aM"/></g><path d="M270 425V286h367v-1631H270
```

Comment:
For consistency, should you add also _Passport broker signature_ to the yellow part?

Reply:
done